frankenphp.go

```diff
@@ -720,13 +720,6 @@ func go_sapi_flush(rh C.uintptr_t) bool {
 		return true
 	}
 
-	if r.ProtoMajor == 1 {
-		if _, err := r.Body.Read(nil); err != nil {
```

Comment:
It could be nice to keep (and fix) this if the full-duplex mode isn't enabled, but it looks impossible to detect if full duplex is enabled or not using the request object.